src/cobra/core/gene.py

```diff
@@ -271,11 +272,13 @@ def remove_from_model(
             the model.
 
         """
-        warn(
+        logger.warning(
             "Use cobra.manipulation.remove_genes instead to remove genes "
             "from the model."
         )
-        warn("Use cobra.manipulation.delete_model_genes to simulate knockouts.")
+        logger.warning(
+            "Use cobra.manipulation.delete_model_genes to simulate knockouts."
+        )
```

Comment:
Maybe we should execute on this deprecation rather than fixing the code here?

Reply:
You mean just raise a warning and do nothing? If not, what do you mean? I don't understand this comment.